### PR TITLE
Add more companies to Match Group

### PR DESCRIPTION
**Pull request by rugk**
Sources:
* Statista, Dating Services Report 2019, https://de.statista.com/statistik/studie/id/40367/dokument/online-dating-services-outlook, page 31
* privacy policy of the group, see e.g. [the Tinder one](https://policies.tinder.com/privacy/intl/en/) "With other Match Group businesses"
* links on the given websites

### CODE REVIEW

File: companies/match-group.json [hunk 1]
```diff
@@ -36,6 +36,32 @@
...
+        "PetPeopleMeet.com",
+        "BBPeople (bbpeoplemeet.com)",
+        "InterracialPeopleMeet.com",
+        "Chemistry.com",
+        "MarriageMindedPeopleMeet.com",
+        "LatinoPeopleMeet.com",
+        "DemocraticPeopleMeet.com",
+        "RepublicanPeopleMeet.com",
+        "BabyBoomerPeopleMeet.com",
+        "singleparentmeet.com"
```
Comment (mal-tee) [inline companies/match-group.json:64]:
```suggestion
        "singleparentmeet.com",
```
😄

**Comment by baltpeter** [comment]
Wow. That's a lot of dating sites. Thanks for collecting those.

From looking at a couple of privacy policies, unfortunately, I don't think we can have them all in one record, though. :/ The ones I looked at all had specific physical and email addresses. So, I'm afraid they'll probably all need separate records.

We already have records for some of them like [LoveScout24](https://www.datenanfragen.de/company/lovescout24) and [Plentyoffish](https://www.datenanfragen.de/company/mtch-technology).

**Comment by rugk** [comment]
Yeah, it's a little crazy company structure I guess, given that they have aquired companies and so on, but they certainly all belong to the Match Group.

>  I don't think we can have them all in one record, though
> So, I'm afraid they'll probably all need separate records.

Maybe we cannot (just) add them in one record, however, they may certainly be listed in the Match Group as they belong there. Because AFAIK if you ask a parent company of some companies via GDPR, they still have to fulfill your request and send you your own data.
Also Tinder was listed there and has it's own entry too. So maybe you should just choose which company to ask when you ask for your data. :thinking:

**Comment by baltpeter** [comment]
From our experience, it's better to use the privacy mail to the particular service you are using and our schema doesn't support multiple emails per record. As this is not a common case, it's easier and makes the code less complex to just have multiple records.

In this particular case, it might get quite confusing to have that many "MTCH Technology Services Limited" records, though. I would thus propose that we make an exception here and don't put the legal name but rather the service name in the `name` field and then have "MTCH Technology Services Limited" as the first line in the address. That should clear things up.

> Also Tinder was listed there and has it's own entry too.

I only see [one entry](https://www.datenanfragen.de/company/match-group) for Tinder or am I misunderstanding you?

**Comment by rugk** [comment]
> I only see one entry for Tinder or am I misunderstanding you?

Ah correct. Sorry, I thought you already had a separate file just like for Plentyoffish etc.

**Comment by mal-tee** [comment]
Soo, what's the status here? :D

Does every service have its own email? If some share the "privacyinquiries@gotinder.com" mail, then we could keep them in this record and ditch the others. 

Feel free to create records for the others, would be very much appreciated. Otherwise, you put them in a new Issue so someone can create records on another day. :D

**Comment by rugk** [comment]
I'm sorry, but I currently do not have the time to create all of these records. Feel free to create an issue of them or so.

**Comment by mal-tee** [comment]
Okay, will do. Thank you!

**Comment by mal-tee** [comment]
Done. I will close this PR now.

**Pull request closed without merging**